.travis.yml

```diff
@@ -99,7 +99,15 @@ script:
       echo "Skipping script build step in Coverity branch/job";
     else
       travis_retry travis_wait 50 \
-          mvn clean verify --projects janusgraph-${MODULE} ${ARGS};
+          mvn clean verify --projects janusgraph-${MODULE} -Pcoverage ${ARGS};
+    fi
+
+after_success:
```

Comment:
Are we going to have the same issue here as with Coverity, in that it will upload to Codecov for each of the individual jobs, i.e., a dozen times per build? Is it worth running this manually once in a while the same way as we're doing with Coverity, or do we want it to run automatically to get the report for every PR (to get incremental change in code coverage), or at least have one less thing to think about?

Will Codecov start rate-limiting the submissions?